pkgs/by-name/li/libphidget22/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  automake,
+  libusb1,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libphidget22";
+  version = "0-unstable-2024-04-11";
+  src = fetchurl {
```

```suggestion
  version = "0-unstable-2024-04-11";

  src = fetchurl {
```
Nit: Convention is to put an empty line between `version` and `src`